src/equations/covariant_shallow_water_split.jl

```diff
@@ -103,7 +102,7 @@ end
                    0.5f0 * (vcon_ll[2] + vcon_rr[2]) * mass_flux)
 end
 
-# Non-symmetric part of entropy-conservative flux
+# Non-symmetric part of entropy-conservative flux. Can be used in both surface and volume.
```

Comment:
Maybe add that the treatment of the bottom topography is consistent with the non-conservative flux of Wintermeyer for flat manifolds?